src/sass/_core.scss

```diff
@@ -1,11 +1,13 @@
 // Generate Duration && Delay
 [data-aos] {
   @for $i from 1 through 60 {
+    :host [data-aos-duration='#{$i * 50}'] &,
```

Comment:
> This is an experimental technology

`:host` is not supported by IE, Edge and Safari.

https://developer.mozilla.org/en-US/docs/Web/CSS/:host()#Browser_compatibility